.github/workflows/tests.yml

```diff
@@ -3,6 +3,7 @@ name: 'Rails Lint and Test'
 on:
   workflow_dispatch:
   push:
+    branches: [ master ]
```

Comment:
Does it need to run on other stable branches as well? `sac-master` maybe? But it seems it is just a delayed state of `master` anyways.